pkg/fix.go

```diff
@@ -92,6 +93,11 @@ func (o *FixOptions) Run() error {
 		}
 	}
 
+	var customPrefixes []string
+	if o.CustomClientset != "" {
+		customPrefixes = strings.Split(o.CustomClientset, ",")
```

Comment:
some sanity checking would be good, e.g. ensure there are no empty values, ensure the values end in a `/`

Reply:
Some basic checks have been added